superset/viz.py

```diff
@@ -1231,6 +1232,39 @@ def get_data(self, df):
         return df.to_dict(orient='records')
 
 
+class ChordViz(BaseViz):
+
+    """A Chord diagram"""
+
+    viz_type = "chord"
+    verbose_name = _("Directed Force Layout")
+    credits = '<a href="https://github.com/d3/d3-chord">Bostock</a>'
+    is_timeseries = False
+
+    def query_obj(self):
```

Comment:
You need to override `query_obj` and it has to return a query object, which is essentially a Python `dict` with key/values that are expected by the `query` method of the connectors. Until this is better documented, you can look at the connector's Datasource `query` method to get a sense for it.